src/sentry/grouping/strategies/__init__.py

```diff
@@ -6,7 +6,6 @@ def _import_all():
     strategy_modules = [
         'message',
         'security',
-        'threads',
```

Comment:
It makes sense to exclude threads for grouping. It would be good to maybe put a comment here or somewhere to make it clear that this was not oversight.

Reply:
Threads were not removed from grouping, it's just that the code moved.